app/code/Magento/Customer/Block/Widget/Telephone.php

```diff
@@ -176,4 +176,14 @@ public function isRequired()
         return $this->_getAttribute(self::ATTRIBUTE_CODE) ? (bool)$this->_getAttribute(self::ATTRIBUTE_CODE)
             ->isRequired() : false;
     }
+
+    /**
+     * Get the telephone validation attribute
+     *
+     * @return string
+     */
+    public function getValidationClass()
```

Comment:
Why we need this method? We can use directly $this->getAttributeValidationClass('telephone').

Reply:
I pushed I fix to merge the two similar methods, as we are using this class just for the telephone widget template, I think the validation method can be specific and already set the validation class required to the telephone attribute.
What do you think?

Reply:
I pushed a fix to merge the two similar methods, as we are using this class just for the telephone widget template, I think the validation method can be specific and already set the validation class required to the telephone attribute.
What do you think?